intl_tel_widget/static/src/js/phone_widget_frontend.js

```diff
@@ -0,0 +1,22 @@
+var input = document.querySelector(".intl_tel_widget");
+var phone_output = document.querySelector(".phone_output");
+var phone_value = document.querySelector(".phone_value");
+
+var iti = window.intlTelInput(input, {
+    utilsScript: "https://cdn.jsdelivr.net/npm/intl-tel-input@17.0.3/build/js/utils.js",
```

Comment:
What do you think about using `/intl_tel_widget/static/lib/intl-tel-input/js/utils.js` instead of loading from an external site?

Reply:
Simply because the local lib didn't work for me i tried it more than once and it just wasted my time i think its outdated ... If you could get it to work it will be great i will if i had the time.

Reply:
Thanks for the clarification, I understand, that decision though is for this repo maintainer

Reply:
I think it would be better to use local lib files, so you can replace old libraries to new ones.